components/library/botan/botan.p5m

```diff
@@ -14,293 +14,401 @@
 # Copyright 2021 Andreas Wacknitz
 #
 
-set name=pkg.fmri value=pkg:/library/security/botan@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+# Drop static libraries:
+<transform path=usr/lib/.*\.a -> drop>
```

Comment:
Isn't this already handled by `transforms/archive-libraries-drop`?

Reply:
it seem that the transforms/archive-libraries-drop is not referenced by make-rules/ips.mk

Reply:
> it seems that the transforms/archive-libraries-drop is not referenced by make-rules/ips.mk

Interesting.  It looks like it is completely unused.  Anyway, you could use it by adding `PKGMOGRIFY_TRANSFORMS += $(WS_TOP)/transforms/archive-libraries-drop` to your `Makefile`.  Up to you.

Reply:
We cannot drop static libraries automatically because we have rare cases where we only have static libs. So, the rule of thumb is to drop them if shared libs are available.

Reply:
> We cannot drop static libraries automatically because we have rare cases where we only have static libs. So, the rule of thumb is to drop them if shared libs are available.

Or we can do it this way: #9236